service/experiment-requests.yaml

```diff
@@ -18,6 +18,10 @@
 #
 # You can run "make presubmit" to do basic validation on this file.
 # Please add new experiment requests towards the top of this file.
+- experiment: 2021-05-22-symccafl
+  description: "Symcc-AFL first test"
+  fuzzers:
+    - symcc_afl
```

Comment:
you probably want to compare this against other fuzzers, wanna include these in this list
https://github.com/google/fuzzbench/blob/master/service/core-fuzzers.yaml

Reply:
I was wondering about this, because in the documentation https://google.github.io/fuzzbench/getting-started/adding-a-new-fuzzer/#requesting-an-experiment for requesting an experiment it says  *so you only need to include fuzzers that you’ve modified in a meaningful way* - I have only modified `symcc_afl` - thus I thought it would simply grab knowledge from earlier experiments for comparison purposes?

Reply:
Sure, that is fine, in this case, you will have to wait till experiment ends to see meaningful results. basically at end of experiment, we will merge results with a previous experiment. that should be ok as well (i think with one fuzzer, report has some issue, but we will see).

Reply:
okay - if it gives us live results and avoids potential issues then let's just specify all fuzzers - I put in all the core ones now.